sdk/tests/core/test_az_mqtt5_topic_parser.c

```diff
@@ -15,18 +15,20 @@
 
 #define TEST_SAMPLE_FORMAT                                                               \
   "test/" _az_MQTT5_TOPIC_PARSER_CLIENT_ID_KEY "/" _az_MQTT5_TOPIC_PARSER_SERVICE_ID_KEY \
-  "/" _az_MQTT5_RPC_EXECUTOR_ID_KEY "/" _az_MQTT5_TOPIC_PARSER_COMMAND_ID_KEY "/endoftest"
+  "/" _az_MQTT5_RPC_EXECUTOR_ID_KEY "/" _az_MQTT5_TOPIC_PARSER_NAME_KEY                  \
```

Comment:
Should there be any checks in the code preventing telemetry and command keys from being in the same topic format?

Reply:
That would not happen since the calls to `test_az_mqtt5_topic_parser_replace_tokens_in_format_success` would be called from the RPC api where the user would know that the `{name}` they are replacing is a command name and the same goes for telemetry.